unlock_all_dlcs/mod.txt

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Hello World!",
-    "description": "Testing Lua",
-    "author": "",
-    "contact": "",
+    "name": "All DLC Unlocker",
+    "description": "Unlocks all DLCs",
+    "author": "tsk.xz",
+    "contact": "@tsk.xz",
```

Comment:
Just dont put the words dlc / unlocker or anything like that in your mod.txt. There are mods like Cheater kicker just have a list of words that will ban if it contains it.
Example of the script list of words that will result a ban
![image](https://github.com/8fn/DLC-Unlocker-PD2/assets/47666341/32c66e35-e5d9-4034-a3c8-70cfe06ab961)
![image](https://github.com/8fn/DLC-Unlocker-PD2/assets/47666341/587e0860-173b-4661-ad2d-48b37b60ef61)

Reply:
I updated the mod.txts, deleted 'unlock' and 'unlocks' and changed 'dlc' to 'Downloadable Content' (same meaning, different words) is that good enough or should I delete all DLC mentions completely?